packages/agents/sdk/.nycrc.json

```diff
@@ -1,6 +1,13 @@
 {
   "extends": "@istanbuljs/nyc-config-typescript",
-  "exclude": ["src/index.ts", "src/sdkStableSwap.ts", "src/lib/errors/*.ts", "src/lib/helpers/*.ts", "test/**/*.ts"],
+  "exclude": [
+    "src/index.ts",
+    "src/sdkPool.ts",
+    "src/sdkStableSwap.ts",
```

Comment:
OK for now but we shouldn't be skipping this.

Reply:
Yep, coverage is low here since we have a bunch of wrapper functions for simple readTxns. Those will be extracted to peripheral services.